### PR TITLE
Increase Travis' stability in end to end tests

### PR DESCRIPTION
**Pull request by Droogans**

### CODE REVIEW

**Comment by Droogans** [comment]
@parlarjb Please merge this after reviewing and discussing it with the relevant parties. I am certain it will restore order to Travis test runs in the future.

**Comment by Droogans** [comment]
I think by switching away from sharding the tests between two firefox instances back to just one for everything, we may reduce the chances of the tooltips tests failing for no reason.

**Comment by Droogans** [comment]
Alright, squashing all of this and doing one last run.

File: grunt-tasks/options/shell.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = function (grunt) {
         },
 
         screenshotsClone: {
-            command: ['[ ${ghToken} ] &&',
+            command: ['[ ! -z "${ghToken}" ] &&',
```
Comment (parlarjb) [inline grunt-tasks/options/shell.js]: To make sure I'm parsing it properly, "if the `ghToken` is not a null string" ?

Comment (Droogans) [inline grunt-tasks/options/shell.js]: Yeah, @hussamd told me it's a better way to test if something is a non-empty string than what I was doing earlier.

**Comment by parlarjb** [comment]
LGTM

File: grunt-tasks/options/shell.js (hunk 1 above)
Comment (CITguy) [inline grunt-tasks/options/shell.js]: `-z` tests for zero length, so `! -z` is a non-zero length value

**Comment by CITguy** [comment]
LGTM

**Pull request merged**